src/Microsoft.Framework.PackageManager/Publish/PublishRuntime.cs

```diff
@@ -43,7 +43,7 @@ public bool Emit(PublishRoot root)
 
             new PublishOperations().Copy(_runtimePath, TargetPath);
 
-            if (PlatformHelper.IsMono)
+            if (!RuntimeEnvironmentHelper.IsWindows)
```

Comment:
Do we want something like `RuntimeEnvironmentHelper.IsUnixLike` (which is `IsLinux || IsDarwin`)?